### DISCUSSION TITLE
Content editing optimization for content over 100k

### DISCUSSION THREAD

**Discussion by gaowanliang** [anyproto/contributors#235] [Any-Block & Anytype-Heart]
I have a Markdown file with more than 100k characters of content. Importing it into AnyType (including headings, tables, etc., excluding images) results in the program being unable to open the file at all and freezing completely. Editing it is also very difficult if you copy and paste it into a new page. The following is the file after being desensitized.

[desensitized.md](https://github.com/user-attachments/files/17145629/desensitized.md)

**Comment by gaowanliang** [comment]
Although Notion has a big lag in editing this file, it can still be edited, while AnyType cannot be operated.

**Comment by fuksman** [comment]
Hello @gaowanliang, thank you for the report. We have created an issue in our internal tracker and will address it in our upcoming releases.
